OBJECT/N_0x5CE2E45A5CE2E45A.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: OBJECT 
+---
+
+## _0x5CE2E45A5CE2E45A
+
+```c
+// 0x5CE2E45A5CE2E45A 
+void _0x5CE2E45A5CE2E45A(Any p0);
```

Comment:
Name seems to be `FORCE_PORTABLE_PICKUP_LAST_ACCESSIBLE_POSITION_SETTING` with 1 param (The pickup id).